packages/rest/src/lib/RequestManager.ts

```diff
@@ -28,7 +30,7 @@ export interface RawFile {
 	/**
 	 * The actual data for the file
 	 */
-	rawBuffer: Buffer;
+	fileData: string | number | boolean | Buffer | ReadStream | ServerResponse;
```

Comment:
Is it readstream or readable from the stream module?

Reply:
The packages specifies a file stream